samples/tcp-echo/tcp-echo-ipv6.yaml

```diff
@@ -0,0 +1,61 @@
+# Copyright 2018 Istio Authors
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+################################################################################
+# tcp-echo service
+################################################################################
+apiVersion: v1
+kind: Service
+metadata:
+  name: tcp-echo
+  labels:
+    app: tcp-echo
+    service: tcp-echo
+spec:
+  ipFamilyPolicy: SingleStack
+  ipFamilies:
+  - IPv6
```

Comment:
this is only to be used for validation that single stack still works when using are doing their testing